app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -8,7 +8,9 @@ def create_in_provider(manager_id, params)
       tower_object = provider_collection(manager).create!(params)
 
       refresh(manager)
-      find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
+      target = find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
+      refresh(target)
```

Comment:
To me this looks like a combination of [`refresh_new_target`](https://github.com/ManageIQ/manageiq/pull/14247) (the oVirt event-based refresh stuff I talked about earlier) and [`native operation workflow`](https://github.com/ManageIQ/manageiq/pull/14405).

Talked to @agrare a little offline and sounds like we're in agreement.

The trick here is that this change is blocking the final Fine build scheduled for next week.  And, I'm not sure there's going to be time to change this all around to try to use any of that.